en/docs/references/use-a-connector-in-your-component.md

```diff
@@ -0,0 +1,57 @@
+# Use a Connector 
+
+## Use a connector in your component design
+When you design your components in the low-code view, you can search and add the connectors displayed in the Marketplace as follows:
+
+1. Open the component you are designing in the VS Code Editor and add a connector.
+    
+    ![Add connector](../assets/img/marketplace/add-connector.png){.cInlineImage-small}
+
+2. Search for the connector you need to add (e.g., World Bank), and click on it to add it.
+
+    ![Select connector](../assets/img/marketplace/select-connector.png){.cInlineImage-small}
+
+If the connector is not available by default, you need to visit the vendor portal, make a payment or provide your Freemium account details if required, and access the required authorization tokens beforehand.
+
+e.g., If you need to use the Azure SQL DB connector, create an Azure account and obtain an authorization token.
+
+
+## Update a connector
+
+If you want to introduce a change to your component and update the connector you published for it, you need to explicitly republish the connector.
+
+### Update a private connector
+
+To update a private connector, you can republish the same connector version as follows:
+
+1. Sign in to the Choreo Console at [https://console.choreo.dev](https://console.choreo.dev).
+2. Navigate to your component and click  **Manage**.
+3. Click **Lifecycle** from the left pane.
+4. Click **Republish Connector**.
+
+
+### Update a public connector 
```

Comment:
Let's change this to update a custom connector with 'Public visibility'

Reply:
If you look at APIs, we don't generally say APIs with public visibility, rather we use the term public APIs. Therefore used the same pattern here too.